python/l1tPh2Nanotables_cff.py

```diff
@@ -55,14 +80,14 @@
     doc = cms.string("GTT Track MET"),
     singleton = cms.bool(True), # the number of entries is variable
     variables = cms.PSet(
-        # pt = Var("pt", float, doc="MET pt"),
-        pt = Var("hwPt() * 0.03125", float, doc = "Track MET"), # as in https://github.com/cms-l1t-offline/cmssw/blob/phase2-l1t-integration-14_0_0_pre3/L1Trigger/L1TTrackMatch/interface/L1TkEtMissEmuAlgo.h#L50
-        hwPt = Var("hwPt()", int, doc = "hardware Pt Track MET"),
-        # phi = Var("phi", float, doc="MET phi"),
-        # hwValid = Var("hwQual() > 0",int, doc = "hardware Missing Et valid bit"),
-        hwValid = Var("hwQual()",bool, doc = "hardware Missing Et valid bit"),
-        # hwVectorSumPt = Var("Et().range()", int, doc = "hardware Missing Et vector sum"),
-        hwPhi = Var("hwPhi", int, doc = "hardware Missing Et phi"),
+        # as in https://github.com/cms-l1t-offline/cmssw/blob/phase2-l1t-integration-14_0_0_pre3/L1Trigger/L1TTrackMatch/interface/L1TkEtMissEmuAlgo.h#L50
+        pt = Var("hwPt() * 0.03125", float, doc = "Track MET"),
+        # as in https://github.com/cms-l1t-offline/cmssw/blob/phase2-l1t-integration-14_0_0_pre3/L1Trigger/L1TTrackMatch/interface/L1TkEtMissEmuAlgo.h#L51
+        phi = Var("hwPhi() * 0.00076699039", float, doc = "Track MET Phi"),
+        et = Var("et", float, "track transverse energy"),
```

Comment:
how does this differ from pt?

Reply:
Doesn't, removed (We'll add the scalar sum in later, when we fix things in CMSSW)

Reply:
But then we could just use pt= et without the hw conversion?

Reply:
but could we then just do `pt = Var("et", float, "track MET")` instead of the hw->float conversion with the fixed scale?